lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -10,16 +10,15 @@
 */
 .nx-html, .nx-body {
   box-sizing: border-box;
-  height: 100%;
```

Comment:
> Are we sure this didn't break the default layout?

Drat, it does. Unfortunately any `height` attribute on the HTML tag cannot be unset in a way that satisfies the requirements for page scrolling so I had to isolate the styles. The consequence of this is that the consumer must specify via class whether the want page or section scrolling, there is no default (though with a bit more work I could make page scrolling the default). https://github.com/sonatype/sonatype-react-shared-components/pull/38/commits/872a96066f24c2cafdd949e2ba3176b704baf4ec

Reply:
> The consequence of this is that the consumer must specify via class whether the want page or section scrolling, there is no default (though with a bit more work I could make page scrolling the default)

I don't see why this would be the case.  Can't you set `height: 100%` on `.nx-html`, and `height: auto` on `.nx-html--page-scrolling`?

Reply:
> > The consequence of this is that the consumer must specify via class whether the want page or section scrolling, there is no default (though with a bit more work I could make page scrolling the default)
> 
> I don't see why this would be the case. Can't you set `height: 100%` on `.nx-html`, and `height: auto` on `.nx-html--page-scrolling`?

This wasn't working yesterday at all, now it does. Then I merged Master and it stopped working again. So went back to the working state with two classes, and then redid the changes and it's working again, I have no idea what that was about, maybe some kind of browser caching: https://github.com/sonatype/sonatype-react-shared-components/pull/38/commits/2dd8d2f2b18e9039f3170dd4efa51da117e36aad